runtime-transaction/src/instructions_processor.rs

```diff
@@ -17,6 +22,36 @@ pub fn process_compute_budget_instructions<'a>(
         .sanitize_and_convert_to_compute_budget_limits()
 }
 
+type RuntimeTransactionInstructionsProcessor =
+    (PrecompileSignatureDetails, ComputeBudgetInstructionDetails);
+impl InstructionProcessor<ProgramIdFlag> for RuntimeTransactionInstructionsProcessor {
+    type OUTPUT = Self;
+
+    #[inline]
+    fn process_instruction(
+        &mut self,
+        flag: &ProgramIdFlag,
+        instruction_index: usize,
+        instruction: &SVMInstruction,
+    ) -> Result<(), TransactionError> {
+        let (signature_details, compute_budget_details) = self;
+        signature_details.process_instruction(&flag.into(), instruction_index, instruction)?;
+        compute_budget_details.process_instruction(&flag.into(), instruction_index, instruction)
+    }
+
+    fn finalize(self) -> Self::OUTPUT {
```

Comment:
it doesn't really do anything tangible, maybe we can remove this extra step in processing